### PR TITLE
Add Mint

### LINKED ISSUE #419: Style Variation: Mint
Captura de ecrã 2023-09-19, às 11 58 22
Captura de ecrã 2023-09-19, às 11 58 15

[mint.json.zip](https://github.com/WordPress/twentytwentyfour/files/12659011/mint.json.zip)

### PR DESCRIPTION
**Pull request by carolinan**
**Description**
Copied from Mint 2: https://github.com/WordPress/twentytwentyfour/issues/419

### CODE REVIEW

**Comment by luminuu** [comment]
I think the fonts are missing as they're referred to Instrument Sans and Jost, @beafialho and @MaggieCabrera have been discussing them in #420

**Comment by luminuu** [comment]
Added the fonts Instrument Sans and Jost to the main `theme.json` file and updated the style variation file a bit.

File: styles/mint.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--instrument-sans)",
+					"fontSize": "var(--wp--preset--font-size--large)",
+					"fontStyle": "normal",
+					"fontWeight": "400"
+				}
+			},
+			"core/site-title": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--instrument-sans)"
```
Comment (MaggieCabrera) [inline styles/mint.json]: if we do this this way it won't work with the rest of the patterns that are using heading/body naming for their fonts. Why are we defining the font on the global theme.json if it's not using it? We should do it on the variation instead. Core will be adding a way for users to add new fonts soon so the theme doesn't need to worry about providing the user with extras

Comment (luminuu) [reply styles/mint.json]: I've compared it with TT3 where all fonts were put in the theme.json. Haven't thought about the heading/body naming. Just added an update.

**Review by MaggieCabrera — APPROVED** [review]
I made the changes to the fonts so they fit the design and work with the patterns

**Pull request merged**